src/Deployer/Task/PlatformConfiguration/NginxTask.php

```diff
@@ -37,13 +37,14 @@ public function configureWithTaskConfig(TaskConfigurationInterface $config): ?Ta
 
         task('deploy:nginx', [
```

Comment:
So now it first places nginx files and after that it runs HMV. Sounds OK to me, but what if the vhost doesn't exist on the node yet? Have you tested that scenario as well?

Reply:
Not yet, but I believe the deploy creates the www.shop.nl folders in the nginx directory aswell, not sure if creating a new vhost will then remove this. Let me see if I can test this somewhere :)

Reply:
Could you verify this or drop this part of the patch? Then we can merge!

Reply:
I'll try and verify this change soon but it's the most important fix of this MR. I'll create a new MR to fix the hypernode settings

Reply:
Just verified this change and it does infact fix my issue when creating an entirely new vhost during deploys when it does not exist yet. Same goed for the deploy:varnish:activate step as this step uses the current folder